src/main/scala/com/gu/inspectorlambda/aws/AWSInspector.scala

```diff
@@ -59,41 +62,25 @@ class AWSInspector(val client: AmazonInspector) extends StrictLogging {
     }
   }
 
-  // These implementations can be deleted once the old bad targets and
-  // templates have been removed.
-  def deleteOldAssessmentTarget(name: String): Unit = {
-    logger.info(s"Searching for old ${name} assessment targets to remove")
-    val assessmentTargets = getAllAssessmentTargets(None)
-    logger.info(s"Found ${assessmentTargets.size} old assessment targets")
-    val matchingAssessmentTargets = assessmentTargets.filter(assessmentTarget => assessmentTarget.getName.equals(name))
-    logger.info(s"Found ${matchingAssessmentTargets.size} old assessment targets with correct name: ${name} ")
-    matchingAssessmentTargets
-      .foreach(assessmentTarget => {
-        logger.info(s"Deleting ${assessmentTarget.getArn}")
-        val deleteAssessmentTargetRequest = new DeleteAssessmentTargetRequest()
-          .withAssessmentTargetArn(assessmentTarget.getArn)
-        client.deleteAssessmentTarget(deleteAssessmentTargetRequest)
-      })
-  }
-
   def getAssessmentTarget(name: String, arn: String): Option[String] = {
     val assessmentTargets = getAllAssessmentTargets(None)
 
     val matchingAssessmentTargets = assessmentTargets.filter(assessmentTarget => assessmentTarget.getName.equals(name))
 
     // delete if arn is not correct
     matchingAssessmentTargets
-      .filter(!_.getResourceGroupArn.equals(arn))
+      .filterNot{at: AssessmentTarget => at.getResourceGroupArn.equals(arn) }
```

Comment:
`.filterNot(_.getResourceGroupArn.equals(arn))` would be better, but we certainly don't want the type here.

As discussed, `.filterNot(at => at.getResourceGroupArn.equals(arn))` is a simple way to appease IntelliJ.